Author: "A. Maitland Bottoms" <bottoms@debian.org>
Date: October 2018

Do not force use of pkg-config upon UHD users.
(but, if user used include(FindPkgConfig) already,
honor that choice)

--- a/host/cmake/Modules/UHDConfig.cmake.in
+++ b/host/cmake/Modules/UHDConfig.cmake.in
@@ -36,7 +36,6 @@
     LIST(APPEND UHD_LIBDIR_HINTS ${UHD_DIR}/lib)
 ENDIF()
 
-INCLUDE(FindPkgConfig)
 IF(PKG_CONFIG_FOUND)
   IF(NOT ${CMAKE_VERSION} VERSION_LESS "2.8.0")
     SET(UHD_QUIET "QUIET")
